spec/controllers/api/saved_jobs_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe Api::SavedJobsController, type: :controller do
+
+  it "will require a user to be signed in to see saved jobs" do
```

Comment:
Honestly having a hard time breaking down these tests,   can we extract the logic into methods that we can reuse between each test.